src/api/app/assets/stylesheets/webui2/colors.scss

```diff
@@ -0,0 +1,17 @@
+$body-bg: #f6f6f6;
+$gray-100: #f8f9fa;
+$gray-150: #f0f0f0;
+$gray-200: #e9ecef;
+
+$main_green: #693;
+$main_blue: #069;
+$main_yellow: #f0ad4e;
```

Comment:
Why `$main_`?

Reply:
Why not? :smile: 
Is not the yellow of bootstrap... 
Do you have a better naming? I will change it if so :+1:

Reply:
> Why not?

Okay I thought it had any significance. Usually it's best to name thing what they are:  `webui2_yellow`

Reply:
I wouldn't use the "webui2" prefix, that's something meant to be temporary. I prefer to have a definitive name for everything as they don't collide with the old stuff.

Reply:
`obs_`? :-)